### PR TITLE
Validation: image-alt-text-duplicated

### PR DESCRIPTION
**Pull request by annmariehitchcock**
This PR addresses the issue: image-alt-text-duplicated errors (duplicate alt text descriptions within an article)
This PR fixes 20 issues across 1 file and is part of ongoing Validation cleanup efforts.
No other content has been edited or added.

### CODE REVIEW

File: aspnet/web-forms/overview/getting-started/hands-on-labs/using-page-inspector-in-visual-studio-2012.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ In this task, you will learn how to use the Page Inspector in the context of an
     *Toggle Inspection Mode button*
 6. In the Page Inspector browser, move the mouse pointer over the page elements. While you move the mouse pointer over any part of the rendered page, the element type is displayed and the corresponding source markup or code is highlighted in the Visual Studio editor.
 
-    ![Inspection mode in action](using-page-inspector-in-visual-studio-2012/_static/image5.png)
+    ![Screenshot showing the Page Inspector window and the Visual Studio editor with the element type displayed and the corresponding source markup is highlighted.](using-page-inspector-in-visual-studio-2012/_static/image5.png)
```
Comment (MeeraDi) [inline aspnet/web-forms/overview/getting-started/hands-on-labs/using-page-inspector-in-visual-studio-2012.md:123]: ...with the element type displayed and the corresponding source markup highlighted.

File: aspnet/web-forms/overview/getting-started/hands-on-labs/using-page-inspector-in-visual-studio-2012.md [hunk 2]
```diff
@@ -212,36 +212,36 @@ In this task, you will use the Page inspector and fix some issues the Photo Gall
 
     Notice that the **#login** style is automatically located in **Site.css** after you click. Moreover, the code is now highlighted.
 
-    ![Selecting the CSS styles](using-page-inspector-in-visual-studio-2012/_static/image16.png "Selecting the CSS styles")
+    ![Screenshot of the Styles tab on the navigation bar the CSS styles for login is selected with the corresponding code highlighted.](using-page-inspector-in-visual-studio-2012/_static/image16.png "Selecting the CSS styles")
```
Comment (MeeraDi) [inline aspnet/web-forms/overview/getting-started/hands-on-labs/using-page-inspector-in-visual-studio-2012.md:215]: Screenshot of the Styles tab, on the navigation bar the CSS styles for login is selected with the corresponding code highlighted.

**Comment by annmariehitchcock** [comment]
Global effort to fix validation errors

@Rick-Anderson 

The Content & Learning team is fixing build validation errors on docs.microsoft.com. This effort will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. I’d very much appreciate your timely review and feedback on these changes I’ve suggested. Thanks!

**Comment by annmariehitchcock** [comment]
Global effort to fix validation errors

@Rick-Anderson

The Content & Learning team is fixing build validation errors on docs.microsoft.com. This effort will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. I’d very much appreciate your timely review and feedback on these changes I’ve suggested. Thanks!

**Pull request merged**